controller/space_test.go

```diff
@@ -402,6 +403,25 @@ func (rest *TestSpaceREST) TestListSpacesOK() {
 	}
 }
 
+func (rest *TestSpaceREST) TestListSpacesOKEmptyResult() {
+	// delete all existing spaces
+	svc, ctrl := rest.SecuredController(testsupport.TestIdentity)
+	_, existingSpaces := test.ListSpaceOK(rest.T(), svc.Context, svc, ctrl, nil, nil, nil, nil)
+	spaceRepository := space.NewRepository(rest.DB)
+	for _, existingSpace := range existingSpaces.Data {
+		rest.T().Log("Deleting existing space with ID=", existingSpace.ID.String())
+		spaceRepository.Delete(svc.Context, *existingSpace.ID)
```

Comment:
I moved the test of empty list retrieval to https://github.com/almighty/almighty-core/pull/1074/files#diff-98914f859911696f3cdd98dd289d00faR1330 instead.